gradle/libs.versions.toml

```diff
@@ -118,6 +119,7 @@ kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-c
 kotlinx-coroutines-guava = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-guava", version.ref = "coroutines" }
 kotlinx-coroutines-test = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-test", version.ref = "coroutines" }
 material = { module = "com.google.android.material:material", version.ref = "material" }
+accompanist = { module = "com.google.accompanist:accompanist-systemuicontroller", version.ref = "systemuicontroller" }
```

Comment:
This name should be more specific like `accompanist-systemuicontroller`

Reply:
Done